README.md

```diff
@@ -1 +1,107 @@
-# eye-of-sauron
\ No newline at end of file
+# eyeofsauron
+
+eyeofsauron is a command-line tool designed to generate the necessary files that makes it super easy to interact with the Sauron API. It completely abstracts away the complexity of authentication and interacting with the GraphQL APIs.
+
+## Features
+
+- Generate necessary files required for interacting with the Sauron API
+- Automatically install required dependencies
+
+## Installation
+
+### Prerequisites
+
+- NodeJS
+
+### Installing eyeofsauron
+
+```bash
+
+npm install -g @gandalfpage/eyeofsauron
+
+```
+
+### Usage
+
+```bash
+
+eyeofsauron generate
+
+```
+
+### Flags
+
+- -f, --file [filename]: Specify the filename of the file to be generated
+- -j, --javascript : Specify if the generated files should be compiled to Javascript.
+
+### Examples
```

Comment:
I don't think this entire example section is necessary. I think most people should just stick to the normal/default section